execute-around/src/main/java/com/iluwatar/execute/around/App.java

```diff
@@ -42,14 +41,11 @@
    */
   public static void main(String[] args) throws IOException {
 
-    new SimpleFileWriter("testfile.txt", new FileWriterAction() {
-
-      @Override
-      public void writeFile(FileWriter writer) throws IOException {
-        writer.write("Hello");
-        writer.append(" ");
-        writer.append("there!");
-      }
-    });
+    FileWriterAction writeHello = writer -> {
```

Comment:
We should add `@FunctionalInterface` annotation to `FileWriterAction`.

Reply:
Done. I did a fix, soft reset, force push since I wanted to fix my email address in the last commit.